java/dagger/internal/codegen/TypeNames.java

```diff
@@ -79,7 +78,6 @@
   static final ClassName MAP_PRODUCER = ClassName.get(MapProducer.class);
   static final ClassName MAP_PROVIDER_FACTORY = ClassName.get(MapProviderFactory.class);
   static final ClassName MEMBERS_INJECTOR = ClassName.get(MembersInjector.class);
-  static final ClassName MEMBERS_INJECTORS = ClassName.get(MembersInjectors.class);
```

Comment:
See `MemberSelect.noOpMembersInjector()`

Reply:
Ok, right. 
So can I remove static `injectMembers` from `MembersInjectors`. It's seems to be not used.